### PR TITLE
Collection Landing page Mobile Optimizations

### PR DESCRIPTION
**Pull request by danthewolfe**
Added media-query stylesheets for page breaks at 480 and 768.
Reduce branding size, expand title height to accommodate long titles on small screens (<768).
Change modals to cover more of the screen, with less margin/padding and use a smaller font on small screens.

### CODE REVIEW

File: app/assets/stylesheets/_media-medium.css.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  .title-bar {
+    max-height: none;
+  }
+
+  .modal.in {
+    z-index: 6000;
+  }
+
+  .modal-body > div {
+    height: 100% !important;
```
Comment (jaronkk) [inline app/assets/stylesheets/_media-medium.css.scss]: Is there a way to get these to work without the !important call?

Comment (danthewolfe) [inline app/assets/stylesheets/_media-medium.css.scss]: Yes, it would require moving some css out of the react components.

**Comment by jaronkk** [comment]
:+1: 

I'm okay with this.  I'd like to avoid uses of !important if possible, but some of that can wait until we settle on our design a little more.

**Comment by danthewolfe** [comment]
I've managed to remove most of the !importants, but the ones that remain are over-riding css that is computed within react components.

File: app/assets/stylesheets/ui-customizations.css.scss [hunk 2]
```diff
@@ -1063,4 +1071,27 @@ body.collection {
...
+.navbar-nav {
+  margin: 0;
+}
+
+
 @import "_material-design-overwrites.scss";
+
+// media-query stylesheet
+@import "_media-medium.css.scss";
+@import "_media-small.css.scss";
```
Comment (jaronkk) [inline app/assets/stylesheets/ui-customizations.css.scss]: @danthewolfe Does rewriting the import in the recommended format work?

Comment (danthewolfe) [inline app/assets/stylesheets/ui-customizations.css.scss]: No, it does not. Hound is incorrect in this case.

**Comment by jonhartzler** [comment]
:+1:

**Pull request merged**